package.json

```diff
@@ -76,7 +76,8 @@
     "netlify-cli": "12.0.2",
     "postcss": "8.4.17",
     "prettier": "2.7.1",
-    "ts-jest": "29.0.3"
+    "ts-jest": "29.0.3",
+    "vue": "^3.2.40"
```

Comment:
**question:** we don't use vue in this project, so why are we adding it exactly?

Reply:
@coderbyheart @codesee/tracker is in the devDependencies and that has vue as a dev dependency. https://www.npmjs.com/package/@codesee/tracker/v/0.51.1